docs/cri/config.md

```diff
@@ -323,21 +324,6 @@ version = 2
   # It generates a self-sign certificate unless the following x509_key_pair_streaming are both set.
   enable_tls_streaming = false
 
-  # tolerate_missing_hugetlb_controller if set to false will error out on create/update
-  # container requests with huge page limits if the cgroup controller for hugepages is not present.
-  # This helps with supporting Kubernetes <=1.18 out of the box. (default is `true`)
-  tolerate_missing_hugetlb_controller = true
-
-  # ignore_image_defined_volumes ignores volumes defined by the image. Useful for better resource
-	# isolation, security and early detection of issues in the mount configuration when using
-	# ReadOnlyRootFilesystem since containers won't silently mount a temporary volume.
-  ignore_image_defined_volumes = false
-
-  # netns_mounts_under_state_dir places all mounts for network namespaces under StateDir/netns
-  # instead of being placed under the hardcoded directory /var/run/netns. Changing this setting
-  # requires that all containers are deleted.
-  netns_mounts_under_state_dir = false
-
```

Comment:
Any reason why this was removed and added later in the config? Since these are part of the `[plugins."io.containerd.grpc.v1.cri"]`, if defined after the next table i.e `[plugins."io.containerd.grpc.v1.cri".x509_key_pair_streaming]`, it wont be parsed correctly.  Ideally the key_pair_streaming table should be kept after this [here](https://github.com/containerd/containerd/pull/8515/files#diff-91d0a4c61f6d3523b5a19717d1b40b5fffd7e392d8fe22aed7c905fe195b8902R431)

Reply:
Updated. I was not aware that this could result in a parsing error since `[plugins."io.containerd.grpc.v1.cri".x509_key_pair_streaming]` was already defined in the middle of `[plugins."io.containerd.grpc.v1.cri"]` section. Like yoou suggested I have now moved the `x509_key_pair_streaming` section to the end.